monitor.js

```diff
@@ -2,15 +2,24 @@
  * Monitor
  * Pokemon Showdown - http://pokemonshowdown.com/
  *
- * Various utility functions to make sure PS is running healthy.
+ * Various utility functions to make sure PS is running healthily.
  *
  * @license MIT license
  */
 'use strict';
 
 const FS = require('./fs');
 
+/**
+ * @class TimedCounter
```

Comment:
They seem to be the exception when `noImplicitAny` is set to `true`, since Typescript will automatically assume that the property will be of type `any[]` for whatever reason. [I made a pullreq to the Typescript wiki to add documentation for @property declarations mentioning this](https://github.com/Microsoft/TypeScript-wiki/pull/141)